src/browser/rvm/utils.ts

```diff
@@ -21,7 +21,7 @@ let isFlushScheduled: boolean = false;
 let totalBytes: number = 0;
 let timer: NodeJS.Timer = null;
 
-function flushConsoleMessageQueue(): void {
+export function flushConsoleMessageQueue(): void {
```

Comment:
This is an calling an async operation `sendToRVM` but does not expose any way to wait or react once that operation is complete.

At the moment there is no guarantee that this finishes.

Reply:
I made this way jankier but it now blocks (without affecting the ignoring-async-flush behaviour for existing logs).